source/php/Submission.php

```diff
@@ -145,6 +145,7 @@ public function submit()
                     '', $_SERVER['SERVER_NAME']) . '>' : 'no-reply@' . preg_replace('/www\./i', '',
                     $_SERVER['SERVER_NAME']);
         }
+        $fromHelsingborg = _x( 'Helsingborgs Stad', 'email: autoreply from name', 'modularity-form-builder' ) . ' <no-reply@' . preg_replace('/www\./i', '', $_SERVER['SERVER_NAME']) . '>';
```

Comment:
Get WordPress configuration home url. Use parse_url or filter_var to get url domain.